src/pages/Homes.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+const Home = () => {
+  const homeStyle = {
+    padding: '20px 40px',
+    lineHeight: '1.5em',
+  };
+
+  const paragraphStyle = {
+    textAlign: 'justify',
+  };
+
+  return (
+    <div style={homeStyle}>
+      <h1 style={{
+        fontSize: '2.5rem',
+        fontWeight: '600',
+        marginBottom: '2rem',
+        lineHeight: '1em',
+        color: '#0c2c4c',
+      }}
+      >
+        Welcome to our page!
+      </h1>
+      <p style={paragraphStyle}>
+        {/* eslint-disable-next-line max-len */}
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Facere, quod accusamus. Tempore officia repellendus consectetur voluptatem a, iste, obcaecati natus nulla quis distinctio eum quae fuga? Eveniet aliquid ullam nesciunt, voluptate illum voluptates dolor animi temporibus! Aut consequuntur exercitationem ducimus, rem numquam hic officia illo qui, illum praesentium minus quis doloribus, assumenda ratione. Qui maxime fugiat iusto optio ducimus cum neque, amet iste architecto praesentium unde corrupti tempore, ut, nihil laudantium labore? Ut suscipit quae magni corporis beatae nam harum atque aspernatur distinctio cumque quibusdam dignissimos ratione sint ea, aut, amet tenetur, totam quo. Exercitationem ipsum quam deserunt neque veniam.
+      </p>
+      <p style={paragraphStyle}>
+        {/* eslint-disable-next-line max-len */}
```

Comment:
- [ ] OPTIONAL
You have **disabled ESLint checks** on parts of your code. Please enable **all** ESLint checks on your code and try modifying that portion of your code to adhere to ESLint standards. This will ensure a robust and clean code.